.swiftformatignore

```diff
@@ -0,0 +1 @@
+**Package.swift
```

Comment:
So that linked version is actually malformed. The intention is that the comment is on its own line. The formatting rules in all our repos except for a few swift-openapi-* repos currently allow this. Rather than changing the formatting rules for those repos I thought it was less disruptive to copy the `.swiftformatignore` file from `swift-openapi-runtime` and ignore the manifest altogether.